### PR TITLE
Solve #3

### PR DESCRIPTION
**Pull request by denics**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ The default values for the variables are set in `defaults/main.yml`:
 # defaults file for cups
 
 # Allow remote access
+cups_serveralias: "*
```
Comment (robertdebock) [inline README.md]: The closing quote (") is missing.

File: defaults/main.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 # Allow remote access
 cups_listen: "localhost:631"
+cups_serveralias:
```
Comment (robertdebock) [inline defaults/main.yml]: Any parameter must have a value, try `"*"` for example.

**Review by robertdebock — COMMENTED** [review]
Thanks for your PR, please review and modify it.

**Comment by denics** [comment]
I solved the two issues, sorry, probably an issue with my keyboard? I do not really understand how I could have missed all " :) sorry for wasting your time

**Pull request merged**